code/modules/antagonists/superheroes/buzzon/buzzon_equipment.dm

```diff
@@ -0,0 +1,294 @@
+		return
+
+	var/sword_location = get_turf(linked_sword)
+	var/owner_location = get_turf(owner)
+	if(get_dist(owner_location, sword_location) > 10)
+		to_chat(owner,"<span class='warning'>[linked_sword] is too far away!</span>")
+	else
+		sword_cooldown = world.time + 20
+		if(isliving(linked_sword.loc))
+			var/mob/living/current_owner = linked_sword.loc
+			current_owner.dropItemToGround(linked_sword)
+			to_chat(current_owner, "<span class='warning'>[linked_sword]'s small rocket engine suddenly activates and rips it out of your hand!</span>")
+		linked_sword.throw_at(owner, 10, 2)
+
+/obj/item/clothing/head/hooded/bee_hood/full //It's not a helmet because I want tackles and flashbangs to fuck you up
+	name = "cybernetic bee helmet"
+	desc = "A cybernetic helmet attached to a suit. All hail the Queen!"
+	icon_state = "bee_full"
+
+	body_parts_covered = HEAD
+	clothing_flags = THICKMATERIAL
+	flags_inv = HIDEFACE|HIDEHAIR|HIDEFACIALHAIR|HIDEEARS|HIDEEYES|HIDESNOUT
+
+/obj/item/clothing/head/hooded/bee_hood/full/cryo
+	icon_state = "bee_full_cryo"
+
+	cold_protection = HEAD
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+
+/obj/item/clothing/head/hooded/bee_hood/full/equipped(mob/user, slot, initial)
+	. = ..()
+	if(!ishuman(user))
+		return
+
+	if(slot != ITEM_SLOT_HEAD)
+		return
+
+	var/mob/living/carbon/human/buzzon = user
+
+	var/datum/atom_hud/hud = GLOB.huds[DATA_HUD_DIAGNOSTIC_ADVANCED]
+	hud.add_hud_to(buzzon)
+	ADD_TRAIT(buzzon, TRAIT_DIAGNOSTIC_HUD, CLOTHING_TRAIT)
+	ADD_TRAIT(buzzon, TRAIT_ROBOTIC_FRIEND, CLOTHING_TRAIT) //BuzzOn is the best roboticist ever so bots don't attack him even if emagged
+
+/obj/item/clothing/head/hooded/bee_hood/full/dropped(mob/user)
+	. = ..()
+	if(!ishuman(user))
+		return
+	var/mob/living/carbon/human/buzzon = user
+
+	var/datum/atom_hud/hud = GLOB.huds[DATA_HUD_DIAGNOSTIC_ADVANCED]
+	hud.remove_hud_from(buzzon)
+	REMOVE_TRAIT(buzzon, TRAIT_DIAGNOSTIC_HUD, CLOTHING_TRAIT)
+	REMOVE_TRAIT(buzzon, TRAIT_ROBOTIC_FRIEND, CLOTHING_TRAIT)
+
+/obj/item/melee/beesword/buzzon
+	name = "The Stinger"
+	desc = "A sting of giant bee on a handle that became the fastest sword in the world thanks to high-tech components."
+	icon_state = "buzzon_sword"
+	inhand_icon_state = "buzzon"
+	worn_icon_state = "buzzon"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	slot_flags = ITEM_SLOT_BACK
+	block_chance = 10 //It's about agility, not about sheer force
+	///If this var is higher than world.time this means that the owner is still in the blocking stance and has a high chance to block the attack
+	var/parry_time = 0
+
+/obj/item/melee/beesword/buzzon/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(parry_time >= world.time)
+		parry_time = 0
+		final_block_chance += 85
```

Comment:
oh, c'mon, give 'em a full +=100 (remember that the force of the attacking weapon and its AP can reduce the final block chance, hence why I recommend going to 110 instead of 100)